Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -69,8 +69,9 @@ def family
         end
       end
 
+      # Supported CPU instructions
       def flags
-        @flags ||= cpuinfo[/^(flags|Features).*/, 0]&.split
+        @flags ||= cpuinfo[/^(flags|Features)\s*: (.*)/, 2]&.split
```

Comment:
```suggestion
        @flags ||= cpuinfo[/^(?:flags|Features)\s*: (.*)/, 1]&.split
```

Reply:
Yes, I like non-capturing groups too and am OK with this suggestion. @sjackman, do you see any possible issues with this suggestion?